### PR TITLE
feat: adding marketing url to the algolia index

### PR DESCRIPTION
**Pull request by alex-sheehan-edx**
## Description

Adding marketing URL to the algolia index to provide users a method of reaching the B2C course details page.

## Post-review

Squash commits into discrete sets of changes

### CODE REVIEW

File: enterprise_catalog/apps/catalog/algolia_utils.py [hunk 1]
```diff
@@ -66,6 +66,7 @@
         'searchable(subjects)',
         'first_enrollable_paid_seat_price',
         'original_image_url',
+        'marketing_url',
```
Comment (taliaedX) [inline enterprise_catalog/apps/catalog/algolia_utils.py:70]: doesn't this also need to be in the Algolia Fields list or no?

**Review by taliaedX — COMMENTED** [review]
Can't tell if it's missing from a spot or fine as is

File: enterprise_catalog/apps/catalog/algolia_utils.py (hunk 1 above)
Comment (alex-sheehan-edx) [reply enterprise_catalog/apps/catalog/algolia_utils.py:70]: weeeeeeee thx

**Review by taliaedX — APPROVED** [review]
lgtm!

**Pull request merged**